src/Ide/Cradle.hs

```diff
@@ -1,862 +0,0 @@
-{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Here rests my biggest contribution to HIE. 
RIP

Reply:
The most beautifully documented module ⚰️